tests/test_utils/test_sft.py

```diff
@@ -8,77 +8,78 @@
 from pyfstat.utils import get_sft_as_arrays, plot_spectrogram
 
 
-@pytest.fixture
-def timestamps_for_test():
-    # Set up SFTs with arbitrary gaps
-    return {
-        "H1": 1800 * np.array([1, 2, 4, 6, 8, 10]),
-        "L1": 1800 * np.array([1, 3, 5, 7, 9, 12]),
+@pytest.fixture(scope="module", params=["nogaps", "gaps"])
+def data_for_test(tmp_path_factory, request):
+    Tsft = 1800
+    ts_base = np.arange(start=1, stop=7 * Tsft, step=Tsft, dtype=int)
+    timestamps = {
+        "H1": ts_base,
+        "L1": ts_base,
     }
-
-
-@pytest.fixture
-def data_for_test(tmp_path, timestamps_for_test):
+    gaps = request.param
+    if gaps == "gaps":
+        for ifo in timestamps.keys():
+            timestamps[ifo] = np.delete(timestamps[ifo], [2, 4])
```

Comment:
Adding a non-integer fraction of `Tsft` to one of the gaps, because codecov noticed that the `remainder` logic of the function wasn't being tested:
```suggestion
            timestamps[ifo][-1] += 100
```